packages/jest-each/src/bind.js

```diff
@@ -203,9 +196,8 @@ const getMatchingKeyPaths = title => (matches, key) =>
 const replaceKeyPathWithValue = data => (title, match) => {
   const keyPath = match.replace('$', '').split('.');
   const value = getPath(data, keyPath);
-  const valueType = getType(value);
 
-  if (PRIMITIVES.has(valueType)) {
+  if (typeof value !== 'symbol' && isPrimitive(value)) {
```

Comment:
is printing symbol anyhow better in pretty-format than default?

Reply:
Ah I did this because String.replace blows up with symbols

<img width="444" alt="screenshot 2019-01-25 at 17 59 48" src="https://user-images.githubusercontent.com/5610087/51763709-0c666d80-20cb-11e9-8211-872e4285a443.png">

Reply:
Also can't explicitly call `value.toString()` in the string replace because `toString` doesn't exist on `null` or `undefined` - the joys of JS 😝

Reply:
You can always do `String(value)`, that should fix the symbol problem as well
```
> String(Symbol('A'))
< "Symbol(A)"
```

Reply:
Good idea @thymikee seems to fix it xD